packages/flutter/lib/src/material/divider.dart

```diff
@@ -0,0 +1,33 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+import 'theme.dart';
+
+class Divider extends StatelessComponent {
+  const Divider({ Key key, this.height: 16.0, this.indent: 0.0, this.color }) : super(key: key);
```

Comment:
IMHO, it's better to have the height assert in the constructor and have the widget be non-const.  Having the constructor be const limits what we can do with this class in the future.